src/CoreShop/Bundle/CoreBundle/Migrations/Version20200224160300.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace CoreShop\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Pimcore\Migrations\Migration\AbstractPimcoreMigration;
+
+class Version20200224160300 extends AbstractPimcoreMigration
```

Comment:
Why do you need two migrations doing the same (in different bundles)?

Reply:
cause of separation of bundles. When you install full-suite CoreShop, the IndexBundle is not installed, only the CoreBundle. If you only use the IndexBundle, you don't have the CoreBundle installed.... And since it's just migrations, I am happy with the duplication.